test/unit/mdc-toolbar/foundation.test.js

```diff
@@ -0,0 +1,676 @@
+/**
```

Comment:
For all of these tests, please refrain from testing any internal methods/properties, e.g. `calculations_`. Instead, test the component's _behavior_ by verifying that things work as expected by calling handlers, using adapter interactions, etc.